client/src/pages/Profile/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -19,20 +27,28 @@ export default function ProfilePhoto(): JSX.Element {
     },
   });
 
+  useEffect(() => {
+    setPhoto(loggedInUser.profile_photo_url);
+  }, [loggedInUser.profile_photo_url]);
```

Comment:
Going forward, I would suggest to use camelCase as opposed to snake_case. This is JavaScript convention